tests/test_download_firmware.py

```
@@ -3251,3 +2995,106 @@ def test_cleanup_old_versions_scandir_error(self, mock_exists, downloader, mocke
+            patch.object(
+                downloader.cache_manager,
+                "get_repo_directories",
+                return_value=[active_dir],
+            ),
+            patch.object(
+                downloader,
+                "_download_prerelease_assets",
+                return_value=([], [], False),
+            ),
+        ):
+            results, failed, latest, summary = (
```

Ruff (0.15.10)

[warning] Unpacked variable `results` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)